Packs/Orca/Integrations/Orca/Orca.py

```diff
@@ -54,7 +54,7 @@ def get_alerts_by_filter(self, alert_type: Optional[str] = None, asset_unique_id
 
         return alerts
 
-    def get_all_alerts(self, first_fetch: Optional[str],  fetch_informational: bool = False) -> List[Dict[str, Any]]:
+    def get_all_alerts(self, first_fetch: Optional[str], fetch_informational: bool = False) -> List[Dict[str, Any]]:  # pylint: disable=E1136 # noqa: E501
```

Comment:
It would be preferred not to ignore this error.

Reply:
Something is wrong with the linter.
It is saying that **first_fetch** is not subscribable even though I am not subscribing it, that's why I ignored this error